.murdock

```diff
@@ -9,7 +12,7 @@ export DLCACHE_DIR=${DLCACHE_DIR:-~/.dlcache}
 NIGHTLY=${NIGHTLY:-0}
 RUN_TESTS=${RUN_TESTS:-${NIGHTLY}}
 
-DWQ_ENV="-E BOARDS -E APPS -E NIGHTLY -E RUN_TESTS"
+DWQ_ENV="-E BOARDS -E APPS -E NIGHTLY -E RUN_TESTS -E TOOLCHAIN"
```

Comment:
@cladmi and I weren't really sure if this is required

Reply:
From what I understand now, it should not be necessary.

The compile tasks including toolchain will be listed by https://github.com/RIOT-OS/RIOT/blob/a8c53ec387b2cc8cfb7da5b65311dc2f97d52feb/.murdock#L125-L127

And set from the argument in https://github.com/RIOT-OS/RIOT/blob/a8c53ec387b2cc8cfb7da5b65311dc2f97d52feb/.murdock#L136-L139

Reply:
I removed this change, so let's see how murdock might react.

Reply:
Seems to work regardless :-)